From 33e5504fbb5e5e2ff44023c0a22dfb668ff8b10f Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Subject: Add new lock-obj-pub for powerpc64le-unknown-linux-gnu.

* src/syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h: New.
* src/Makefile.am (lock_obj_pub): Add.

--

Here is a arch-specific lock-obj header file for little-endian 64-bit
powerpc.

Debian-bug-id: 762322
---
 src/Makefile.am                                    |  1 +
 .../lock-obj-pub.powerpc64le-unknown-linux-gnu.h   | 25 ++++++++++++++++++++++
 2 files changed, 26 insertions(+)
 create mode 100644 src/syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h

diff --git a/src/Makefile.am b/src/Makefile.am
index 10810a6..def0f45 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -57,6 +57,7 @@ lock_obj_pub = \
         syscfg/lock-obj-pub.mipsel-unknown-linux-gnu.h      \
         syscfg/lock-obj-pub.powerpc-unknown-linux-gnu.h     \
         syscfg/lock-obj-pub.powerpc64-unknown-linux-gnu.h   \
+	syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h \
         syscfg/lock-obj-pub.s390x-ibm-linux-gnu.h           \
         syscfg/lock-obj-pub.sh4-unknown-linux-gnu.h         \
         syscfg/lock-obj-pub.sparc-unknown-linux-gnu.h       \
diff --git a/src/syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h b/src/syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h
new file mode 100644
index 0000000..79073d4
--- /dev/null
+++ b/src/syscfg/lock-obj-pub.powerpc64le-unknown-linux-gnu.h
@@ -0,0 +1,25 @@
+## lock-obj-pub.powerpc64le-unknown-linux-gnu.h
+## File created by gen-posix-lock-obj - DO NOT EDIT
+## To be included by mkheader into gpg-error.h
+
+typedef struct
+{
+  long _vers;
+  union {
+    volatile char _priv[40];
+    long _x_align;
+    long *_xp_align;
+  } u;
+} gpgrt_lock_t;
+
+#define GPGRT_LOCK_INITIALIZER {1,{{0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0, \
+                                    0,0,0,0,0,0,0,0}}}
+##
+## Local Variables:
+## mode: c
+## buffer-read-only: t
+## End:
+##
-- 
2.1.0

